homework.rb

```diff
@@ -1,6 +1,14 @@
-VARIANT = 0
+VARIANT = 7
 
 def generate
   # put your solution here (delete the line below)
-  rand(1000000).to_s + '-' + rand(1000000).to_s
+  letters = []
+ 
+  2.times do |i|
+  	i = ('A'..'Z').to_a.sample(3).join
+  	letters << i
+  end
+ 
+  letters[0] + '-' + rand(1000000..9999999).to_s + '-' + letters[1]
```

Comment:
you interpolate result + call + result... not ruby way.

move create rand number to var and interpolate it.